### PR TITLE
Created halodoc.md

### PR DESCRIPTION
**Pull request by manee2k6**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: adopters/organizations/halodoc.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## **Halodoc**
+
+[Halodoc](https://www.halodoc.com/) is a secure health-tech platform with a mission to simplifying access to healthcare by connecting millions of patients with licensed doctors, insurance, labs, and pharmacies in one simple mobile application. Halodoc’s innovative technology, nimble services, and patient focus enable a host of solutions including 24/7 doctor tele consultation via chat, voice or video; medicine purchase & delivery; lab services at home; and strong customer support.
+
+Halodoc is the 2018 Forbes Indonesia Choice Award winner and Galen Growth’s 2018 Most Innovative HealthTech Startup in Asia, a testimony to a team of compassionate, innovative, trustworthy and agile people who take ownership of their work in building the most trusted digital healthcare company. 
+
+### **Why we explored litmus**
+
+We wanted a tool that we could leverage to test the resiliciency of multi k8s cluster workloads in private cloud,
```
Comment (ksatchit) [inline adopters/organizations/halodoc.md:9]:
```suggestion
We wanted a tool that we could leverage to test the resiliency of multi k8s cluster workloads in a private cloud, 
```

**Review by ajeshbaby — APPROVED** [review]

File: adopters/organizations/halodoc.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+## **Halodoc**
+
+[Halodoc](https://www.halodoc.com/) is a secure health-tech platform with a mission to simplifying access to healthcare by connecting millions of patients with licensed doctors, insurance, labs, and pharmacies in one simple mobile application. Halodoc’s innovative technology, nimble services, and patient focus enable a host of solutions including 24/7 doctor tele consultation via chat, voice or video; medicine purchase & delivery; lab services at home; and strong customer support.
+
+Halodoc is the 2018 Forbes Indonesia Choice Award winner and Galen Growth’s 2018 Most Innovative HealthTech Startup in Asia, a testimony to a team of compassionate, innovative, trustworthy and agile people who take ownership of their work in building the most trusted digital healthcare company. 
+
+### **Why we explored litmus**
+
+We wanted a tool that we could leverage to test the resiliciency of multi k8s cluster workloads in private cloud, 
+after evaluating different tools of similar flavour, we wanted to excercise our chaoas experiments using Litmus, it has the following benefits.
```
Comment (ksatchit) [inline adopters/organizations/halodoc.md:10]:
```suggestion
after evaluating different tools of similar flavour, we wanted to exercise our chaos experiments using Litmus, as it has the following benefits.
```

File: adopters/organizations/halodoc.md (hunk 1 above)
Comment (manee2k6) [reply adopters/organizations/halodoc.md]: updated spell resiliency and "in a private"

File: adopters/organizations/halodoc.md (hunk 2 above)
Comment (manee2k6) [reply adopters/organizations/halodoc.md]: updated spell exercise , chaos and "as it has the following"

**Comment by ksatchit** [comment]
Thank you @manee2k6 :pray: for sharing your adoption story with the litmus community!

**Pull request merged**